pkg/stanza/operator/input/tcp/tcp_test.go

```diff
@@ -78,7 +79,7 @@ func tcpInputTest(input []byte, expected []string) func(t *testing.T) {
 		cfg := NewConfigWithID("test_id")
 		cfg.ListenAddress = ":0"
 
-		op, err := cfg.Build(testutil.Logger(t))
+		op, err := cfg.Build(testutil.Logger(t), component.TelemetrySettings{})
```

Comment:
It might not matter yet but can we use `componenttest.NewNopTelemetrySettings()` instead of the zero value?